src/text-field-components.tsx

```diff
@@ -129,17 +135,14 @@ export const FieldContainer: React.FC<FieldContainerProps> = ({
                 [styles.disabled]: disabled,
             })}
             onClick={(e) => {
-                // workaround for the multiline case, where the textarea has a margin. We want to focus
-                // the input when the user clicks anywhere in the container (like in the label)
-                if (multiline) {
-                    e.currentTarget.querySelector('textarea')?.focus();
-                }
+                // We want to focus the input when the user clicks anywhere in the container (like in the label or the prefix)
+                e.currentTarget.querySelector(multiline ? 'textarea' : 'input')?.focus();
```

Comment:
If TextField had a prefix and you clicked on it, it was not focusing the input element